### ISSUE TITLE
Unhandled Exception: type 'HttpException' is not a subtype of type 'FileInfo'

### ISSUE THREAD

**Issue by GuruJohn** [renefloor/flutter_cached_network_image#210]
I think I get this error the second time a image tries to load. If the first time was a 404.
It still seems to work correctly and show the errorWidget: or placeholder: it just also raises this error.

It makes for very noisy logs if you have a lot of images that might 404.

```
E/flutter (27538): [ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: type 'HttpException' is not a subtype of type 'FileInfo'
E/flutter (27538): #0      BaseCacheManager.getFile (package:flutter_cache_manager/src/cache_manager.dart:120:11)
E/flutter (27538): <asynchronous suspension>                            
E/flutter (27538): #1      CachedNetworkImageState._animatedWidget (package:cached_network_image/src/cached_image_widget.dart:270:12)
E/flutter (27538): #2      CachedNetworkImageState.build (package:cached_network_image/src/cached_image_widget.dart:196:12)
E/flutter (27538): #3      StatefulElement.build (package:flutter/src/widgets/framework.dart:3922:27)
E/flutter (27538): #4      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3836:15)
E/flutter (27538): #5      Element.rebuild (package:flutter/src/widgets/framework.dart:3657:5)
E/flutter (27538): #6      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3819:5)
E/flutter (27538): #7      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3961:11)
E/flutter (27538): #8      ComponentElement.mount (package:flutter/src/widgets/framework.dart:3814:5)
E/flutter (27538): #9      Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #10     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #11     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4973:14)
E/flutter (27538): #12     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #13     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5079:32)
E/flutter (27538): #14     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #15     MultiChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5079:32)
E/flutter (27538): #16     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #17     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #18     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4973:14)
E/flutter (27538): #19     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #20     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #21     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4973:14)
E/flutter (27538): #22     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #23     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #24     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4973:14)
E/flutter (27538): #25     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #26     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #27     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3847:16)
E/flutter (27538): #28     Element.rebuild (package:flutter/src/widgets/framework.dart:3657:5)
E/flutter (27538): #29     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3819:5)
E/flutter (27538): #30     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:3961:11)
E/flutter (27538): #31     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3814:5)
E/flutter (27538): #32     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #33     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #34     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3847:16)
E/flutter (27538): #35     Element.rebuild (package:flutter/src/widgets/framework.dart:3657:5)
E/flutter (27538): #36     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3819:5)
E/flutter (27538): #37     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3814:5)
E/flutter (27538): #38     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #39     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #40     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:4973:14)
E/flutter (27538): #41     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3053:14)
E/flutter (27538): #42     Element.updateChild (package:flutter/src/widgets/framework.dart:2856:12)
E/flutter (27538): #43     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3847:16)
E/flutter (27538): #44     Element.rebuild (package:flutter/src/widgets/fr
```

**Comment by blaneyneil** [comment]
you beat me by 5 minutes.  i'm using firebase storage for images and have an awful lot of:

type 'HandshakeException' is not a subtype of type 'FileInfo'

```
cache_manager.dart in BaseCacheManager.getFile at line 120
Called from: <asynchronous suspension>
cached_image_widget.dart in CachedNetworkImageState._animatedWidget at line 270
cached_image_widget.dart in CachedNetworkImageState.build at line 196
```

**Comment by renefloor** [comment]
Yes I am aware. It is being discussed here: https://github.com/renefloor/flutter_cache_manager/issues/99

**Comment by renefloor** [comment]
fixed in 1.1.1
